### ISSUE TITLE
Detect ALT text in the Cover block

### ISSUE THREAD

**Issue by stevejburge** [publishpress/PublishPress-Checklists#322]
> I am adding the checklist pro plugin to a client site and they are testing it with different Gutenberg blocks. Image, media+text, and gallery all work well detecting if the images have alt text or not. We are having a hard time with the cover block. I am adding the alt text inside the media Library modal and inserting the image into the cover block, but the "cannot not publish" restriction still persists.

**Comment by stevejburge** [comment]
Can no longer replicate
